app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -337,8 +370,13 @@ def build_filtered_item_list(action_type, filter_type)
     filtered_item_list
   end
 
+  def schedule_db_backup_or_automate(schedule)
+    %w(db_backup automation_request).include?(schedule.sched_action[:method])
+  end
+
   def determine_filter_type_and_value(schedule)
-    if schedule.sched_action && schedule.sched_action[:method] && schedule.sched_action[:method] != "db_backup"
+    if schedule.sched_action && schedule.sched_action[:method] && !schedule_db_backup_or_automate(schedule)
```

Comment:
missing `&&` here?

(It's been a while so not sure what this could afect, noticed during https://github.com/ManageIQ/manageiq-ui-classic/pull/5427 review)